web/src/components/dashboard/HideableTasks.tsx

```diff
@@ -0,0 +1,71 @@
+import { Icon } from "@/components/njwds/Icon";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useRoadmap } from "@/lib/data-hooks/useRoadmap";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { getSectionNames } from "@/lib/utils/helpers";
+import { LookupOperatingPhaseById } from "@businessnjgovnavigator/shared/operatingPhase";
+import { ReactElement } from "react";
+import { Button } from "../njwds-extended/Button";
+import { Step } from "../Step";
+import { SectionAccordion } from "./SectionAccordion";
+
+export const HideableTasks = (): ReactElement => {
+  const { userData, update } = useUserData();
+  const { roadmap } = useRoadmap();
+  const { Config } = useConfig();
+
+  if (!LookupOperatingPhaseById(userData?.profileData.operatingPhase).displayHideableRoadmapTasks) {
+    return <></>;
+  }
+
+  const handleToggleClick = () => {
+    if (!userData) return;
+    update({
+      ...userData,
+      preferences: {
+        ...userData.preferences,
+        isHideableRoadmapVisible: !userData.preferences.isHideableRoadmapVisible,
+      },
+    });
+  };
+
+  return (
+    <div className="margin-top-7" data-testid="hideableTasks">
+      <div className="fdr margin-bottom-105">
+        <h2>{Config.dashboardDefaults.upAndRunningTaskHeader}</h2>
+        <div className="mla">
+          <Button style="narrow-light" onClick={handleToggleClick}>
+            <div className="fdr fac">
+              <Icon>{userData?.preferences.isHideableRoadmapVisible ? "visibility_off" : "visibility"}</Icon>
+              <span className="margin-left-05 line-height-sans-2">
+                {userData?.preferences.isHideableRoadmapVisible
+                  ? Config.dashboardDefaults.hideTaskText
+                  : Config.dashboardDefaults.showTaskText}
+              </span>
+            </div>
+          </Button>
+        </div>
+      </div>
+      <hr className="margin-bottom-3 margin-top-0" aria-hidden={true} />
+
+      {userData?.preferences.isHideableRoadmapVisible ? (
+        <>
+          {getSectionNames(roadmap).map((section) => (
+            <SectionAccordion key={section} sectionType={section}>
+              {roadmap &&
+                roadmap.steps
+                  .filter((step) => step.section === section)
+                  .map((step, index, array) => (
+                    <Step key={step.stepNumber} step={step} last={index === array.length - 1} />
+                  ))}
+            </SectionAccordion>
+          ))}
+        </>
+      ) : (
+        <div className="text-base">
+          {roadmap?.tasks.length} {Config.dashboardDefaults.hiddenTasksText}
```

Comment:
typically what we do for phrases that have a dynamic component like a count is to use `${}` notation in the config string itself. For example, take a look at `hiddenOpportunitiesHeader` in the config. 

We want to write the whole phrase including the part that will be substituted for the number. That way, Jess and Ce can rearrange the phrase as needed without a code change. If we change the value of `hiddenTasksText` to `"${count} Hidden Tasks"` then they could, for example, change the content later on to read as `"You have ${count} Hidden Tasks"` without needing us to change any code. And we use `templateEval` to insert the value for `count` into the string.